gtk4/src/subclass/widget.rs

```diff
@@ -550,8 +557,15 @@ unsafe extern "C" fn widget_compute_expand<T: WidgetImpl>(
     let mut vexpand: bool = from_glib(*vexpand_ptr);
```

Comment:
... these are a problem because you read uninitialized memory here depending on whether `Xexpand_set` is true or not.

Reply:
Ah my bad. I should have took a coffee before looking at this.